.env

```diff
@@ -1,3 +1,4 @@
 TRANSLATION_SERVICE_URL=http://localhost:8080/cql/translator
 INPUT_CQL=./cql
-OUTPUT_ELM=./src/output-elm
\ No newline at end of file
+OUTPUT_ELM=./src/output-elm
+REACT_APP_MEASUREMENT_ID=G-JK8FGR8HVG
```

Comment:
Can we add some documentation on this property in the README?

Reply:
I added a section to the README.